src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +68,55 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block {
```

Comment:
The separated classes should not be in the same java file with Address class. You need to create a new java file for each class.

Reply:
Also note that the header comment for each class is required. (A description of what this class is about)